packages/@react-spectrum/table/test/TestTableUtils.test.tsx

```diff
@@ -227,6 +256,58 @@ describe('Table ', function () {
       expect(onSortChange).toHaveBeenCalledTimes(3);
       expect(onSortChange).toHaveBeenLastCalledWith({column: 'foo', direction: 'descending'});
     });
+
+    it('highlight selection with modifier key', async function () {
+      render(<TableExample allowsResizing selectionStyle="highlight" />);
+      let tableTester = testUtilFakeTimer.createTester('Table', {root: screen.getByTestId('test')});
+      tableTester.setInteractionType(interactionType);
+
+      await tableTester.toggleRowSelection({row: 2, selectionBehavior: 'replace'});
+      // Called multiple times because highlight selections will change selection on focus unless you use
+      // a modifier key
+      if (interactionType === 'keyboard') {
+        expect(onSelectionChange).toHaveBeenCalledTimes(2);
+        expect(new Set(onSelectionChange.mock.calls.at(-1)[0])).toEqual(new Set(['Foo 1', 'Foo 3']));
+      } else {
+        expect(onSelectionChange).toHaveBeenCalledTimes(1);
+        expect(new Set(onSelectionChange.mock.calls.at(-1)[0])).toEqual(new Set(['Foo 3']));
+      }
+
+      await tableTester.toggleRowSelection({row: 'Foo 4', selectionBehavior: 'replace'});
+      if (interactionType === 'keyboard') {
+        expect(onSelectionChange).toHaveBeenCalledTimes(3);
+        expect(new Set(onSelectionChange.mock.calls.at(-1)[0])).toEqual(new Set(['Foo 1', 'Foo 3', 'Foo 4']));
+      } else {
+        expect(onSelectionChange).toHaveBeenCalledTimes(2);
+        expect(new Set(onSelectionChange.mock.calls.at(-1)[0])).toEqual(new Set(['Foo 3', 'Foo 4']));
+      }
+    });
+  });
+
+  describe('long press, fake timers', () => {
+    installPointerEvent();
+    let testUtilFakeTimer = new User({interactionType: 'touch', advanceTimer: jest.advanceTimersByTime});
+    beforeAll(function () {
+      jest.useFakeTimers();
+    });
+
+    afterEach(function () {
+      act(() => jest.runAllTimers());
+      jest.clearAllMocks();
+    });
+
+    it('highlight selection should switch to selection mode on long press', async function () {
```

Comment:
is this the same describe and test as just below?

Reply:
urgh, must have duplicated in a merge conflict resolution, thanks for catching